Rejestrator/main/main.ino

```diff
@@ -15,12 +19,6 @@ CameraController cameraController;
 SDCardController sdCardController;
 BluetoothController bluetoothController;
 
-void writeVideoConfigToMemory()
```

Comment:
that function went to BicFunctions so that bluetoothController can use it to save user config